FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Chen Yi-Wei on 2021/9/3.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal struct FeedImagesMapper {
```

Comment:
The convention in this codebase is to omit the default access level keyword of `internal`, so it should be removed.
```suggestion
struct FeedImagesMapper {
```

Reply:
`FeedImageMapper` is defined as a struct, so we can instantiate it as `FeedImageMapper()`. However, all its methods are static - it has no instance methods or properties.

It may be confusing for other developers to be able to instantiate a class but then not be able to do anything with it.

As you'll see in the following lectures, you can make the initializer private in such cases since the `FeedImageMapper` is just a namespace for static methods, with no instance-specific behavior.

Or you can make it an enum with no cases, which can never be instantiated!

Both solutions will disallow instantiation at compile time so that `FeedImageMapper()` won't compile ✅

Reply:
Hi Penna,
thanks for your review,
it's really very helpful,
I was wondering why not use struct in this case in the lecture,
your suggestion answered my question,
thanks a lot!